src/parser/monk/mistweaver/integrationTests/example.test.ts.snap

```diff
@@ -5137,7 +5132,7 @@ Array [
        even if the channel continues past the buff.
     </React.Fragment>,
     "stat": <React.Fragment>
-      6,963 average mana saved per Mana Tea cast
+      0 average mana saved per Mana Tea cast
```

Comment:
Did you expect this number to change? I assumed the functionality of the modules wasnt being changed at all here

Reply:
Looking more into this something did break

Reply:
Looking into it more it looks like it was wrong even before that... somehow... 🤔